src/service/growthService.js

```
@@ -0,0 +1,13 @@
+import axiosInstance from "./axiosInstance";
+
+class GrowthService {
+  static async getDob() {
+    try {
+      return axiosInstance.get("/growth/dob");
+    } catch (error) {
+        throw error;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)